hardware/simulation/sim_build.mk

```diff
@@ -80,6 +69,6 @@ test1:
 # 	make -C $(SOC_DIR) sim-clean SIMULATOR=$(SIMULATOR)
 # 	make -C $(SOC_DIR) sim-run SIMULATOR=$(SIMULATOR) INIT_MEM=0 USE_DDR=1 RUN_EXTMEM=1 TEST_LOG=">> test.log"
 
-NOCLEAN+=-o -name "system_tb.v"
-NOCLEAN+=-o -name "system_top.v"
-NOCLEAN+=-o -name "iob_soc_tb.cpp"
+#NOCLEAN+=-o -name "system_tb.v"
```

Comment:
In i2s i started using simulation/src to store files that are not to be deleted or generated at setup time.

Use this way pls and delete NOCLEAN.